### PR TITLE
Densify and generalize

### PR DESCRIPTION
**Pull request by tschie**
Shows how to densify and generalize a geometry using the GeometryEngine. No local data required.

### CODE REVIEW

**Comment by tschie** [comment]
@john0005 Can you please review? Oddly, occasionally it crashes and I haven't been able to figure out why. Seems like a timing issues... maybe with spatial references? Obviously can't merge with it like that but it should be okay for a code review.

File: src/main/java/com/esri/samples/geometry/densify_and_generalize/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+<p>To densify and generalize a polyline:</p>
+
+<ol>
+    <li>Use the static method <code>GeometryEngine.densify(polyline, maxSegmentLength)</code> to densify the polyline
+    . The resulting polyline will add points along the line so that there are no points greater than <code>maxSegmentLength</code> from the next point.</li>
+    <li>Use the static method <code>GeometryEngine.generalize(polyline, maxDeviation, true)</code> to generalize the 
+    polyline. The resulting polyline will have points or shifted from the line to simplify the shape. None of these points can 
+    deviate farther from the original line than <code>maxDeviation</code>. The last parameter, 
+    <code>removeDegenerateParts</code>, will clean up extraneous parts if the geometry is multi-part.</li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/geometry/densify_and_generalize/README.md]: Would put, `removeDegenerateParts  which is set to true`.

**Review by john0005 — APPROVED** [review]
Looks good.

**Comment by tschie** [comment]
Added a note in the readme. Also, the crash issue was due to me using a test build which wasn't working. Works fine against release.

**Pull request merged**